### PR TITLE
Add rename friend

### PR DESCRIPTION
**Pull request by bloomyminded**
Command and specs for renaming an exisiting friend.

### CODE REVIEW

**Comment by bloomyminded** [comment]
I guess the rebase I did included all of your previous changes.  Is this typical?  I have another branch with some fixes for Rubocop errors but it also includes these commits.

**Comment by JacobEvelyn** [comment]
Thanks for the work! I just added some comments to your pull request. Let me know if you have any questions about how to modify your code to address them!

**Comment by bloomyminded** [comment]
Thanks for the feedback.  I'll revise when I get a break today

**Comment by JacobEvelyn** [comment]
Just a few more very small comments and then this looks good to merge! Thanks so much for all the hard work here.

**Comment by bloomyminded** [comment]
Excellent.  Thanks for your patience and suggestions.  I'm pretty green with this stuff.

File: lib/friends/introvert.rb [hunk 1]
```diff
@@ -299,6 +313,13 @@ def elapsed_days
 
     private
 
+    # Replace old name with new name upon friend name change.
+    # @param old_name [String] the old name of the friend
+    # @param new_name [String] the new name of the friend
+    # @return [Friend] the friend whose name was just changed
+    def replace_old_name(old_name, new_name)
```
Comment (JacobEvelyn) [inline lib/friends/introvert.rb]: Looks like you forgot to remove this method.

File: lib/friends/activity.rb [hunk 2]
```diff
@@ -124,6 +124,15 @@ def highlight_friends(introvert:)
       @description.delete!("\\")
     end
 
+    # Updates a friends old_name to their new_name
```
Comment (JacobEvelyn) [inline lib/friends/activity.rb]: Tiny nitpick: missing an apostrophe in `friends` here.

**Comment by JacobEvelyn** [comment]
This looks really good! Pretty high quality for someone describing himself as "green" :smile:. I was about to merge it when I noticed you forgot to remove an empty method, and a teeny-tiny grammatical thing. Fix those and I promise I'll merge!

File: lib/friends/introvert.rb (hunk 1 above)
Comment (bloomyminded) [inline lib/friends/introvert.rb]: /facepalm

**Comment by JacobEvelyn** [comment]
Looks good, merged! Thanks for doing this. I realized I forgot to ask you to put this in the README so I'll write a quick example there too. I'll release this in version 0.12 soon.

**Comment by JacobEvelyn** [comment]
This is released!

**Pull request merged**